use blood for "Vlad was here" engravings

vultur-cadens opened the pull request:
A hero in vampire form using a finger to engrave scrawls in blood; so should Vlad.

copperwater commented:
Suggestion: the messages only have a chance of becoming scrawled in blood after a certain depth. They can appear very early, after all, and I think it increases the creepiness factor if an engraving you've seen before in dust, and thought little of it, is now scrawled in blood.

Vlad doesn't have to make blood engravings if he doesn't want to; writing with a soft item would do the trick of making a dust one.

vultur-cadens committed:
use blood for some "Vlad was here" engravings

A hero in vampire form using a finger to engrave scrawls in blood;
so can Vlad.  The chance of a blood engraving increases with depth:
engravings before level 10 are all dust; the chance increases
linearly until engravings at level 21 are all in blood.

vultur-cadens commented:
Updated.  The chance of "Vlad was here" being written in blood now increases with depth.  0% chance of a blood engraving at levels 6-9, increasing linearly from level 10 until it's 100% at level 21 and deeper.

NullCGT commented:
I really like that. It helps sell the idea that the dungeon is becoming more threatening the deeper you go, and that perhaps the powers that be are beginning to take you more seriously.
